src/app/portlets/dot-edit-page/layout/dot-edit-layout-grid/dot-edit-layout-grid.component.html

```diff
@@ -2,10 +2,10 @@
 <div [ngGrid]="gridConfig"
      (onDragStop)="OnDragStop()"
      class="dot-edit-layout-grid__grid">
-
-    <div style="background: #b2b2b2; border: 1px solid red;"
-         *ngFor="let container of gridContainers; let i = index"
-         [(ngGridItem)]="container.config">Index: {{i}}
-        <a (click)="removeContainer(i)">Remove</a>
-    </div>
+    <dot-container-selector *ngFor="let container of gridContainers; let i = index"
```

Comment:
I'm not 100% sure, but I think we should wrap this in a div and that one will have all related stuff about the grid, maybe that's the problem with the resize not working correctly.

Reply:
after testing the behavior do not change for adding an extra container